gapis/resolve/command_tree.go

```diff
@@ -372,20 +372,31 @@ func addDrawEvents(ctx context.Context, events *service.Events, p *path.CommandT
 }
 
 func addFrameEvents(ctx context.Context, events *service.Events, p *path.CommandTree, t *commandTree, last api.CmdID) {
-	frameCount, frameStart, frameEnd := 0, api.CmdID(0), api.CmdID(0)
+	frameCount, frameStart := 0, api.CmdID(0)
 	for _, e := range events.List {
 		i := api.CmdID(e.Command.Indices[0])
 		switch e.Kind {
```

Comment:
I'm not sure about removing the use of `FirstInFrame`. Don't we want to use this information if it's provided by the API? For example `eglSwapBuffers` is now annotated with `@StartOfFrame`, so we're actually deriving our start-of-frame here from the end-of-frame event, which is actually derived from the `@StartOfFrame` annotation. Ewww.